initial commit

eduardlizet opened the pull request.

eduardlizet committed:
initial commit

eduardlizet committed:
merging

eduardlizet committed:
login/signup handlebars

thegetoutgirl commented on package.json:
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "handlebars-boilerplate",
+  "name": "node-handlebars-boilerplate",
```
accepting the update

thegetoutgirl commented on package.json:
```diff
@@ -11,7 +11,7 @@
   "license": "ISC",
   "dependencies": {
-    "dotenv": "^8.0.0",
+    "dotenv": "^8.2.0",
```
accepting the update

thegetoutgirl committed:
deleting package.json from GitHub

Keep package.json & node_modules locally, rather than pushing to github, caused some pull request issues/conflicts. Don't have to worry about them until the end (?)

You can do this by adding them to your gitignore file!

thegetoutgirl commented on models/index.js:
```diff
@@ -5,7 +5,7 @@ var path = require("path");
 var basename = path.basename(module.filename);
 var env = process.env.NODE_ENV || "development";
-var config = require(__dirname + "/../config/config.json")[env];
+var config = require(__dirname + "/../config/config.js")[env];
```
Accepting new update

thegetoutgirl committed:
Deleting this --

database in models file needs some updates, but can be moved :)

thegetoutgirl committed:
Deleting bootstrap files

using Materialize now

thegetoutgirl reviewed (approved):
Super!

thegetoutgirl committed:
Keep these files locally

add to gitignore so they don't push
